templates/web/base/admin/users/edit.html

```diff
@@ -1,7 +1,7 @@
 [% IF user.id ~%]
     [% title = tprintf(loc('Editing user %d'), user.id ) ~%]
 [% ELSE ~%]
-    [% title = loc('Add staff user') ~%]
+    [% title = loc('Add ' _ (c.user.is_superuser ? '' : 'staff ') _ 'user') ~%]
```

Comment:
I'm afraid this won't work, because `loc()` needs an entire self-contained string for translation purposes (putting the string in a .po file for people to translate), so it can't deal with concatenations and the like. This would have to be `title = c.user.is_superuser ? loc("Add user") : loc("Add staff user")`

Reply:
It looks like I need to do the same on index.html as well.